src/overlays/actors/ovl_Shot_Sun/z_shot_sun.h

```diff
@@ -10,9 +10,11 @@ typedef void (*ShotSunActionFunc)(struct ShotSun*, GlobalContext*);
 
 typedef struct ShotSun {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x4C];
+    /* 0x014C */ ColliderCylinder collider;
     /* 0x0198 */ ShotSunActionFunc actionFunc;
-    /* 0x019C */ char unk_19C[0xC];
+    /* 0x019C */ Vec3s unk_19C;
+    /* 0x01A2 */ s16 spawnTimer; // Frames until fairy spawns
```

Comment:
generally when an actor only has one timer we just call it "timer" cause it can be implied what it does from its usage. when theres more than one though feel free to name it